app/__init__.py

```diff
@@ -98,7 +105,7 @@ def init_rollbar():
             got_request_exception.connect(rollbar.contrib.flask.report_exception, app)
 
     @app.errorhandler(429)
-    def too_many_requests(e):
+    def too_many_requests(_):
         return json_api(TooManyRequestsError, ErrorSchema), 429
 
     @app.errorhandler(500)
```

Comment:
I remove this in my pr but shouldnt have this for public release. Breaks json schema

Reply:
This is about the 500 error btw